rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt

```diff
@@ -248,6 +248,40 @@ data class ZAddStmt(
     }
 }
 
+data class MultStmt(
+    val target: AssignableExpression,
+    @Derived val dataDefinition: InStatementDataDefinition? = null,
```

Comment:
Can this statement define a new variable?

Reply:
Ops!!! Copy and past issue, but... What is the goal of StatementThatCanDefineData?

Reply:
Operations like clear can implicitly define variables